runtime/service_integ_test.go

```diff
@@ -53,7 +53,7 @@ const (
 	defaultNamespace = namespaces.Default
 
 	containerdSockPath = "/run/containerd/containerd.sock"
-	debianDockerImage  = "docker.io/library/debian:latest"
+	guestDockerImage   = "docker.io/library/alpine:latest"
```

Comment:
While we're at it, let's pin this to a specific version instead of `latest` (i.e. `3.10.1)

Reply:
Will do.